tracee-rules/output.go

```diff
@@ -39,23 +47,45 @@ func setupTemplate(inputTemplateFile string) (*template.Template, error) {
 	}
 }
 
+type jsonExporter struct{}
+
+func (j *jsonExporter) Execute(wr io.Writer, data interface{}) error {
+	serializedObject, err := json.Marshal(data)
+	if err != nil {
+		log.Printf("Error when trying to parse tracee event: %v\n", err)
+		return nil
+	}
+
+	bs := make([]byte, 4)
+	binary.LittleEndian.PutUint32(bs, uint32(len(serializedObject)))
+
+	// ensure data is in the same buffer (instead of writing performing consecutive writes)
+	res := bytes.Join([][]byte{bs, serializedObject}, []byte(""))
```

Comment:
@tomyweiss , we still have an issue here. As long as the io.Writer is directed to the stdout, the JSON objects can be mixed with logs (as in line 55).